guides/_subsections/create-an-automated-rule.md

```diff
@@ -1,64 +1,64 @@
+        summary="Configure the New <code>Automated Rule</code>"
         image-name="2.3.0/create-an-automated-rule-2.png"
         text="
       <p>
-        <b>Name:</b> Enter a name for the new rule. The form will alert you if the name
+        <i>Name:</i> Enter a name for the new rule. The form will alert you if the name
         entered has an invalid format. If the name is already in use then the
         creation will fail and you will need to try again.
       </p>
       <p>
-        <b>Description:</b> Enter an optional description for your rule.
+        <i>Description:</i> Enter an optional description for your rule.
       </p>
       <p>
-        <b>Match Expression:</b> We will fill this field in the next step.
+        <i>Match Expression:</i> We will fill this field in the next step.
       </p>
       <p>
-        <b>Enabled:</b> Enable or disable the rule. If disabled, the rule will not be applied to any targets.
+        <i>Enabled:</i> Enable or disable the rule. If disabled, the rule will not be applied to any targets.
       </p>
       <p>
-        <b>Template:</b> Select an event template or enter a custom event definition. If you are
+        <i>Template:</i> Select an <code>Event Template</code> or enter a custom event definition. If you are
         unsure which to choose, the <i>Continuous</i> template is useful for
-        always-on production monitoring with the <i>continuous</i> recording
+        always-on production monitoring with the <i>Continuous</i> recording
         duration setting, and the <i>Profiling</i> template is useful for
         collecting extra information for troubleshooting a specifically
         identified problem with a fixed recording duration.
       </p>
       "
-      %}  
+      %}
   </li>
   <li>
     <details>
-        <summary>Create your match expression</summary>
+        <summary>Create your <code>Match Expression</code></summary>
         <p>
-            The match expression in a rule definition is a Java-like snippet of code that Cryostat interprets and uses to determine if a rule should be applied to any given target. Match expressions should thus evaluate to a boolean value. The simplest match expressions would be the booleans true or false; if we use true, the rule will apply to every target. The expression has a target object in global scope, with the following form in JSON notation:
+            The <code>Match Expression</code> in a rule definition is a <code>Java-like</code> snippet of code that <b>Cryostat</b> interprets and uses to determine if a rule should be applied to any given target. <code>Match Expressions</code> should thus evaluate to a <code>boolean</code> value. The simplest <code>Match Expressions</code> would be the <code>booleans</code> true or false; if we use true, the rule will apply to every target. The <code>Expression</code> has a target object in global scope, with the following form in <code>JSON</code> notation:
```

Comment:
does "Java-like" need to be in a code block? It's just a descriptor of the code, not an actual entity.